apache/couchdb-fauxton
# [partitioned-dbs] Updates to sidebar

## Overview

Forth in a series of PRs that will be submitted in support of the new user-defined partitioned databases feature (apache/couchdb#1605).

Summary:

Update links in the sidebar to include the selected partition key

## Testing recommendations

**Note:** disregard the error `'partition' parameter is mandatory for queries to this view`. It will be addressed in a separate PR

Execute the steps below in a partitioned db:
- Create a new view `p1` in a new ddoc called `ddocPartitioned`
- Create a new view `g1` in a new ddoc called `ddocGlobal`
  - edit `ddocGlobal` and add `"options": { "partitioned": false }`
  - verify that `ddocPartitioned` and `ddocGlobal` have different icons on the sidebar
- Go back to 'All Documents' and select a partition key at the top. Press Enter to confirm.
- Click on each of the items below and verify 1) page opens correctly and 2) selected partition key is preserved.
  - All Documents
  - Create Document, then cancel to go back to All Documents
  - Run A Query with Mango, then go back to All Documents
  - Design Documents, then back to All Documents
  - `ddocGlobal` > `Metadata`
  - `ddocGlobal` > `g1`
  - `ddocPartitioned` > `Metadata`
  - `ddocPartitioned` > `p1` (it'll show an error)
  - (+) button besides 'Design Documents'
    - New Doc, then cancel
    - New View
    - New Mango Indexes
  - (+) button besides `ddocGlobal`, then click New View
  - (+) button besides `ddocPartitioned`, then click New View

## Related Pull Requests

apache/couchdb#1605

## Checklist
